starfish/image/_filter/util.py

```diff
@@ -136,3 +136,10 @@ def preserve_float_range(array: Union[xr.DataArray, np.ndarray]) -> Union[xr.Dat
     if np.any(array > 1):
         array /= array.max()
     return array.astype(np.float32)
+
+def determine_axes_to_apply_over(is_volume: bool):
```

Comment:
What does it mean to apply over an axis?  Which axes to group over to form something to filter?  The terminology is mostly what I have an issue with.

Also, return type, pls.

Reply:
`split_by`? That's probably the closest to the split-apply-combine terminology that's common in the python ecosystem

Reply:
> split_by? That's probably the closest to the split-apply-combine terminology that's common in the python ecosystem

ok sure that seems reasonable.

also, i think that's perhaps common to the sci-py ecosystem...